README.md

```diff
@@ -22,58 +25,61 @@ You first need to build the cli before building the docker images.
 
 ### Base image
 
-If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you need to rebuild the `containerbase/base` image.
+If you make changes to the [`src`](./src/) folder or the [`Dockerfile`](./Dockerfile), you must:
+
+1. run `pnpm build`
+1. rebuild the `containerbase/base` image
 
 ```sh
+pnpm build
 docker buildx bake
 ```
 
-You can use the following command to ignore remote cache for local testing.
-This will probably speedup local builds.
+You can use the following command to ignore the remote cache for local testing.
+This may speed up your local builds.
 
 ```sh
 docker buildx bake  --set *.cache-from=
 ```
 
 ### Test images
 
-To run one of the tests use the following command, it will run the java tests from [`test/java`](./test/java/).
+To run one of the tests use the following command, it will run the Java tests from [`test/java`](./test/java/).
 
 ```sh
 TAG=java docker buildx bake test
 ```
 
-For other test images checkout [`test`](./test/) folder.
+For other test images see the [`test`](./test/) folder.
 
 ### Distro test images
 
-To run the jammy tests use the following command, it will run the test from [`test/Dockerfile.jammy`](./test/Dockerfile.jammy).
+To run the `noble` tests use the following command, it will run the test from [`test/Dockerfile.noble`](./test/Dockerfile.noble).
```

Comment:
🤔 `jammy` is still a valid target.

Reply:
What do you want?

- both `jammy` and `noble`
- only `jammy`
- only `noble`
- something else

Reply:
both